zeppelin-web/bower

```diff
@@ -1,17 +0,0 @@
-#
```

Comment:
Why removing these files?

So far our frontend build structure did not assume having bower or grunt installed globally, but both got installed locally though maven plugin. 

So how shall we a developer run them now?